packages/editor/src/plugins/drag-drop-bg/index.ts

```diff
@@ -0,0 +1,71 @@
+import { EditorPluginType } from '@editor/types/editor-plugin-type'
+
+import { DragDropBgEditor } from './editor'
+import {
+  type EditorPlugin,
+  type EditorPluginProps,
+  object,
+  string,
+  child,
+  number,
+  boolean,
+  list,
+} from '../../plugin'
+
+export const answerZoneImageProps = child({ plugin: EditorPluginType.Image })
+export const answerZoneTextProps = child({ plugin: EditorPluginType.Text })
```

Comment:
since they are not reused anywhere I'd prefer no extra vars for them (or at least no export)

Reply:
This also goes for the other definitions here. 
In general: `Props` naming confuses me. We usually only use that for React Component Prop Types

Reply:
Done :heavy_check_mark: